src/components/ShopCategory/ShopCategory.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './ShopCategory.scss';
+import { Link } from 'react-router-dom';
+import { sectionsLinks } from '../../helpers';
+
+interface Props {
+  cards: Good[];
+}
+
+export const ShopCategory: React.FC<Props> = ({ cards }) => {
+  const imageWidth = 368;
+  const imageGap = 16;
+  const imageCount = 3;
+
+  return (
+    <div
+      className="Category"
+      style={{
+        width: `${imageWidth * imageCount + (imageGap * (imageCount - 1))}px`,
+      }}
+    >
+      <div className="Category__Title">
+        <p className="Category__Title-text">
+          Shop by category
+        </p>
+      </div>
+      <div className="Category__Container">
+        <ul className="Category__List">
+          {sectionsLinks.map(link => (
+            <li className="Category__Item" key={link.imgUrl}>
```

Comment:
imgUrl отмечено как необязательное. Не стоит использовать необязательные свойства в key.
```suggestion
            <li className="Category__Item" key={link.name}>
```

Reply:
думаю без разницы, какой key будет, линка точно не пересечется

Reply:
++